cmd/cli/kubectl-kyverno/utils/output/table/table.go

```diff
@@ -5,7 +5,7 @@ type Table struct {
 }
 
 func (t *Table) Rows(compact bool) interface{} {
-	if !compact {
+	if compact {
```

Comment:
Either rename to `detailedResults` or negate the bool in the caller.

Reply:
done

Reply:
Thank you.